### PR TITLE
docs: update sveltekit install instructions

### PR DESCRIPTION
**Pull request by vhscom**
Light touch-ups based on development changes in SvelteKit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,8 +202,7 @@ const config = {
   // for more information about preprocessors
   preprocess: preprocess(),
   kit: {
-    // hydrate the <div id="svelte"> element in src/app.html
-    target: '#svelte',
+    adapter: adapter(),
```
Comment (userquin) [inline README.md:205]: is the adapter required? the examples/sveltekit is working, can you check it?

File: README.md [hunk 2]
```diff
@@ -459,7 +459,7 @@ Icons({ compiler: 'svelte' })
 
 Type Declarations
 
-For Svelte Kit, on `src/global.d.ts` file:
+For Svelte Kit, on `src/app.d.ts` file:
```
Comment (userquin) [inline README.md:462]: the examples/sveltekit is working, is this a new change on sveltekit?

Comment (vhscom) [reply README.md:462]: Rich changed the name of the file here: https://github.com/sveltejs/kit/pull/3670. I forgot about he example. If you like, I can regenerate it with this pull. Please let me know.

Comment (userquin) [reply README.md:462]: you should also change and test the example, I've updated it a few days ago removing the target and updating the tsconfig.json file

Comment (vhscom) [reply README.md:462]: I force-pushed a change to rename the `global.d.ts` to `app.d.ts`. Tried to run locally with PNPM but I'm not sure how you guys have the projects set-up. I'll leave it to you as I need to move on.

File: README.md (hunk 1 above)
Comment (vhscom) [reply README.md:205]: It's part of the standard boilerplate. I'd recommend including it in the documentation for context. I don't want to get too much into the weeds here so I'll leave this pull here for you to finish off if you'd like.

Comment (vhscom) [reply README.md:205]: It's part of the standard boilerplate. I'd recommend including it in the documentation for context. I don't want to get too much into the weeds here so I'll leave this pull here for you to polish if you'd like.

File: README.md (hunk 2 above)
Comment (userquin) [reply README.md:462]: `pnpm install` and `pnpm run build`  from root and then `pnpm run dev` from `examples/sveltekit` folder

**Review by userquin — APPROVED** [review]

**Comment by vhscom** [comment]
Thanks folks.

**Pull request merged**